build.gradle

```diff
@@ -999,22 +999,27 @@ project('test:system') {
         dependsOn 'jar'
 
         environment "tier2Type", System.getProperty("tier2Type", "nfs")
-        environment "pravegaOperatorVersion",System.getProperty("pravegaOperatorVersion", "latest")
-        environment "bookkeeperOperatorVersion",System.getProperty("bookkeeperOperatorVersion", "latest")
-        environment "zookeeperOperatorVersion",System.getProperty("zookeeperOperatorVersion", "latest")
+        environment "pravegaOperatorVersion", System.getProperty("pravegaOperatorVersion", "latest")
+        environment "bookkeeperOperatorVersion", System.getProperty("bookkeeperOperatorVersion", "latest")
+        environment "zookeeperOperatorVersion", System.getProperty("zookeeperOperatorVersion", "latest")
         // The below properties are used to to specify the pravega published chart and repository to deploy pravega,bookkeeper & zookkeeper operators using helm docker image properties
-        environment "publishedChartName",System.getProperty("publishedChartName", "pravega")
-        environment "helmRepository",System.getProperty("helmRepository", "pravega")
+        environment "publishedChartName", System.getProperty("publishedChartName", "pravega")
+        environment "helmRepository", System.getProperty("helmRepository", "https://charts.pravega.io")
         // To update config map with desired pravega version
-        environment "desiredPravegaCMVersion",System.getProperty("desiredPravegaCMVersion", "0.9.0")
-        environment "desiredBookkeeperCMVersion",System.getProperty("desiredBookkeeperCMVersion", "0.9.0")
+        environment "desiredPravegaCMVersion", System.getProperty("desiredPravegaCMVersion", "0.9.0")
+        environment "desiredBookkeeperCMVersion", System.getProperty("desiredBookkeeperCMVersion", "0.9.0")
         environment "tlsEnabled", System.getProperty("tlsEnabled", "false")
 
         commandLine './kubernetes/setupTestPod.sh'
     }
 
+    task runFluentBitInstall(type: Exec) {
```

Comment:
What about adding a task to download the logs assuming they are stored in the PVC?